resolwe_bio/processes/junctions/regtools.yml

```diff
@@ -0,0 +1,119 @@
+# =========================================
+# Regtools - Junctions Extract and Annotate
+# =========================================
+---
+
+- slug: junctions-annotate
+  name: Annotate novel splice junctions (regtools)
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/rnaseq:regtools
+  data_name: "Novel Junctions ({{ some input }})"
+  version: 0.0.1
+  type: data:junctions:novel
+  category: analyses
+  flow_collection: sample
+  persistence: CACHED
+  description: >
+    Novel splice junctions in an alignment relative to a reference
+  input:
+    - name: extracted
+      label: Junctions in BED format
+      type: data:junctions:bed
+      description: >
+        Splice junctions previously extracted from an alignment, in BED format. Required if a STAR alignment is not provided.
+      required: false
+    - name: star
+      label: STAR alignment
+      type: data:alignment:bam:STAR
+      description: >
+        STAR alignment, required if a .bed junctions file is not provided. 
+      required: false 
+    - name: gtf
+      label: Reference genome annotation (GTF/GFF3)
+      type: data:annotation
```

Comment:
Does regtools work with both gtf and gff3 format? If not, fix the `type` and change the `name` and `label` to match